src/app/feed/feed-post/feed-post.component.html

```diff
@@ -335,4 +334,59 @@
       </div>
     </div>
   </a>
+  <!-- 
+    This HTML is used on the main post in a thread in order to show detailed info
+    about the interactions with that thread.  TODO: We should probably break the
+    "main" post in a thread into it's own file so it can have more custom styling
+    with less code complexity.
+  -->
+  <div class="feed-post__blue-border px-30px pb-5px" *ngIf="showInteractionDetails">
+    <div class="container w-100 p-0px">
+      <!-- DESKTOP --> 
+      <div class="row no-gutters d-none d-lg-block py-10px fs-15px border-top border-bottom border-color-light-grey cursor-pointer">
+        <div class="d-flex justify-content-between">
+          <span (click)="openRecloutsModal($event, '')">
+            <b>{{ globalVars.abbreviateNumber(post.RecloutCount) }}</b> Reclouts&nbsp;
+          </span>
+          <span (click)="openQuoteRecloutsModal($event, '')">
+            <b>{{ globalVars.abbreviateNumber(post.QuoteRecloutCount) }}</b> Quote Reclouts&nbsp;
+          </span>
+          <span (click)="openLikesModal($event, '')">
+            <b>{{ globalVars.abbreviateNumber(post.LikeCount) }}</b> Likes
+          </span>
+          <span (click)="openDiamondsModal($event)">
+            <b>{{ globalVars.abbreviateNumber(post.DiamondCount) }}</b> Diamonds
+          </span>
+        </div>
+      </div>
+      <!-- MOBILE --> 
+      <div class="row no-gutters d-lg-none py-10px fs-15px border-top border-bottom border-color-light-grey cursor-pointer">
+        <div class="col-6 mb-0" (click)="openRecloutsModal($event, '')">
+          <b>{{ globalVars.abbreviateNumber(post.RecloutCount) }}</b> Reclouts&nbsp;
+        </div>
+        <div class="col-6 mb-0" (click)="openQuoteRecloutsModal($event, '')">
+          <b>{{ globalVars.abbreviateNumber(post.QuoteRecloutCount) }}</b> Quote Reclouts&nbsp;
+        </div>
+      </div>
+      <div class="row no-gutters d-lg-none py-10px fs-15px border-bottom border-color-light-grey cursor-pointer">
+        <div (click)="openLikesModal($event, '')" class="col-6 mb-0">
+          <b>{{ globalVars.abbreviateNumber(post.LikeCount) }}</b> Likes
+        </div>
+        <div (click)="openDiamondsModal($event)" class="col-6 mb-0">
+          <b>{{ globalVars.abbreviateNumber(post.DiamondCount) }}</b> Diamonds
+        </div>
+      </div>
+    </div>
+    <feed-post-icon-row
+      class="mt-2px"
+      *ngIf="showIconRow"
+      [noMaxWidth]="true"
+      [post]="post"
+      [postContent]="postContent"
+      [parentPost]="parentPost"
+      [hideNumbers]="true"
+      [afterCommentCreatedCallback]="afterCommentCreatedCallback"
+      [afterRecloutCreatedCallback]="afterRecloutCreatedCallback"
+    ></feed-post-icon-row>
```

Comment:
The width on this component looks a bit strange IMO.  I would expect the icons in the "main" post to line up with the icons of any parents or children.